setup-ovs.sh

```diff
@@ -63,7 +66,6 @@ if [[ $(nmcli conn | grep -c ovs) -eq 0 ]]; then
       nmcli c delete $(nmcli c show |grep ovs-cnv |awk '{print $1}') || true
   else
       nmcli conn mod brcnv-iface connection.autoconnect yes
-      reboot
```

Comment:
would it be a good idea to add a reload to Network Manager instead of the reboot?
```
systemctl restart NetworkManager.service
```

Reply:
Sure. We don't want the reboot in there anymore, since the bridge/bond needs to be created on every boot now.

There is another issue the customer raised around crio binding to the wrong IP which I'm trying to resolve, so more changes are coming.

Reply:
I understand and welcome these changes, but specifically to this line wouldn't it make sense to restart the NetworkManager service so that the changes are "refreshed" - at least on NetworkManager level? If so, could you add it instead of the reboot cmd?

Reply:
@phoracek do you think it makes sense?

Reply:
I don't think that is needed. We explicitly call `nmcli con up`. Am I missing something?

Reply:
no, I get you. then I'm good